server/sonar-server/src/test/java/org/sonar/server/project/ws/SearchActionTest.java

```diff
@@ -267,6 +274,22 @@ public void verify_response_example() throws URISyntaxException, IOException {
     assertJson(response).isSimilarTo(ws.getDef().responseExampleAsString());
   }
 
+  @Test
+  public void should_return_private_flag() throws URISyntaxException, IOException {
+    OrganizationDto organization = db.organizations().insert();
+    userSession.addPermission(ADMINISTER, organization);
+    ComponentDto privateProject = db.components().insertComponent(newProjectDto(organization).setPrivate(true));
```

Comment:
when rebasing on atlas after I have merged my changes, this should be replaced by `db.components().insertPrivateProject(organization));`